src/transactions/test/InvokeHostFunctionTests.cpp

```diff
@@ -527,6 +527,60 @@ TEST_CASE("Stellar asset contract transfer with CAP-67 address types",
     }
 }
 
+TEST_CASE_VERSIONS("multiple soroban ops in a tx", "[tx][soroban]")
```

Comment:
Isn't this test a duplicate of 'multiple ops are not allowed' test ('multiple ops are not allowed' scenario)? Invalid txs won't be normally admitted to the txset at all, so I'm not sure what are we covering here.

Reply:
Ah I couldn't find an existing one, and wanted to try it with fee bumps as a sanity check.